src/System.Text.Encoding.CodePages/tests/EncodingCodePages.cs

```diff
@@ -539,32 +542,33 @@ public static void TestRoundtrippingSpecificCodepageEncoding(string encodingName
         [MemberData(nameof(CodePageInfo))]
         public static void TestCodepageEncoding(int codePage, string webName, string queryString)
```

Comment:
>TestCodepageEncoding [](start = 27, length = 20)

I would suggest to keep the old code inside this test method not changed and then create a new method for testing the round tripping of the Ascii printable range and another method to test the nls encoder/decoder.

This should be exact doing exact tests you are doing and not repeating any test

Reply:
👍